packages/ui-components/src/components/tooltip/tooltip.tsx

```diff
@@ -103,7 +103,7 @@ export class KvTooltip implements ITooltip {
 						reference={this.getContentElement()}
 						options={this.getOptions()}
 					>
-						<kv-tooltip-text class={{ ...getClassMap(this.customClass) }} text={this.text} />
+						{!isEmpty(this.text) && <kv-tooltip-text class={{ ...getClassMap(this.customClass) }} text={this.text} />}
```

Comment:
Do we want to render the div above if there's no content?
Should this condition be checked earlier?